mcp-server/polaris_mcp/tools/catalog.py

```diff
@@ -106,6 +106,10 @@ def call(self, arguments: Any) -> ToolExecutionResult:
         copy_if_object(arguments.get("query"), delegate_args, "query")
         copy_if_object(arguments.get("headers"), delegate_args, "headers")
 
+        realm = arguments.get("realm")
+        if isinstance(realm, str) and realm.strip():
+            delegate_args["realm"] = realm
```

Comment:
Should we raise an error here if the realm isn't configured in the current MCP server? Either because that realm is invalid or MCP server just didn't configure it. We could have common method for that, as each tool needs it. Not a blocker though. I'm fine with a followup.